subscription-service/src/main/resources/db/migration/V0_11__add_system_attributes_to_subscription_table.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE subscription
+ADD created_at TIMESTAMPTZ NOT NULL,
```

Comment:
as it is not null, it has to be initialized. either, update will fail on existing subscriptions.

Reply:
ok i'll initialize it at 01/01/1970